sql-cli/src/sql_cli/macros/tasks.py.jinja2

```diff
@@ -0,0 +1,31 @@
+{% macro define_task(sql_file, input_tables) -%}
+{{ sql_file.get_variable_name() }} = aql.transform_sql(
+    sql="{{ sql_file.get_relative_target_path() }}",
+    parameters={
+        {%- for parameter in sql_file.get_parameters() %}
+        {%- if parameter in input_tables %}
+        "{{ parameter }}": {{ parameter }},
+        {%- endif %}
+        {%- endfor %}
+    },
+    {%- if sql_file.metadata and "conn_id" in sql_file.metadata %}
+    conn_id="{{ sql_file.metadata.conn_id }}",
+    {%- endif %}
+    op_kwargs={
+        "output_table": Table(
+            name="{{ sql_file.path.stem }}",
+            {%- if "database" in sql_file.metadata or "schema" in sql_file.metadata %}
+            metadata=Metadata(
+                {%- if "database" in sql_file.metadata %}
+                database="{{ sql_file.metadata.database }}",
```

Comment:
I like that we're being explicit at first, even though it may be verbose, it will help us - and the end-users - troubleshooting any issues.